src/core/include/iDynTree/Core/VectorFixSize.h

```diff
@@ -214,57 +209,44 @@ namespace iDynTree
     {
         for(unsigned int i=0; i < VecSize; i++ )
         {
-            this->m_data[i] = 0.0;
+            this->operator[](i) = 0.0;
```

Comment:
Why not calling the ``setZero`` method?

Reply:
Done in https://github.com/robotology/idyntree/pull/709/commits/8fc84cd09206b04f8a69ef4fc859425424608069